### ISSUE TITLE
Validation loss doesn't decrease when training with Lightning.

### ISSUE THREAD

**Issue by andreabonvini** [Lightning-AI/lightning#13724]
Hi everyone,

I’m trying to build a segmentation model with [segmentation models pytorch](https://github.com/qubvel/segmentation_models.pytorch) and Lightning. However, when I try to fit my model I see that the validation loss is always pretty high (even if I plot it in tensorboard, it doesn’t seem to decrease at all, but it just goes up and down). The same doesn’t happen when I train the same model in pure Pytorch. The training loss instead seems to keep decreasing as expected.

You can see my Lightning implementation and try to run the code yourself at [this](https://colab.research.google.com/drive/1jTDkA7N6XOxoVQDNbvrX6r2FvkWsJr0l?usp=sharing) Colab link.
